app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class EmailValidator < ActiveModel::EachValidator
```

Comment:
You should really only have one class per file. You can move validators to `app/validators` see [here](https://stackoverflow.com/questions/42253003/where-should-i-put-the-custom-validators-in-rails-5)

Reply:
This is very useful, thank you!